src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java

```diff
@@ -9,13 +9,40 @@
 public class SelectTestFigureOptionListener implements ActionListener {
 
 	private DriverManager driverManager;
+	private int testNumber;
 
-	public SelectTestFigureOptionListener(DriverManager driverManager) {
+	public SelectTestFigureOptionListener(DriverManager driverManager, int testNumber) {
 		this.driverManager = driverManager;
+		this.testNumber = validateTestNumber(testNumber);
+	}
+
+	private int validateTestNumber(int testNumber) {
+
+		switch (testNumber) {
+
+            case 2:
+				return 2;
+
+			default:
+				return 1;
```

Comment:
Messy code formatting. 
By using enums you could avoid this awkward validation.